src/vm/man/vmadm.1m.md

```diff
@@ -1745,6 +1745,35 @@ tab-complete UUIDs rather than having to type them out for every command.
         create: yes
         update yes (requires zone stop/boot)
 
+    nics.*.pci_slot:
+
+        Specifies the virtual PCI slot that this NIC will occupy. Bhyve places
```

Comment:
I also have a partial branch for this that I never got to work due to VM.js being rather confusing around nics...

Perhaps move the generic explenation of the bus/dev/function, we repeat it nearly verbatem for all *.pci_slot property and just keep the default device number used?

The man page is already pretyt long and just repeating nearly the same text a few times makes editing/reading it harder.